dev/environment

```diff
@@ -5,6 +5,8 @@ WAREHOUSE_ENV=development
 WAREHOUSE_TOKEN=insecuretoken
 WAREHOUSE_IP_SALT="insecure himalayan pink salt"
 
+USERDOCS_DOMAIN="http://localhost:10000"
```

Comment:
NB: This ensures that local deployments link to the local user docs build, rather than `docs.pypi.org` (which is otherwise the default).